src/game/g_utilities.cpp

```diff
@@ -619,25 +619,30 @@ std::vector<int> getMatchingIds(const std::string &name) {
   return pidsVector;
 }
 
-std::string interpolateNametags(std::string input) {
+std::string interpolateNametags(std::string input, const int color) {
   std::string interpolated;
   std::vector<std::string> split = ETJump::StringUtil::split(input, "@");
 
   if (split.size() == 1 || split.size() == 2) {
     return input;
   }
 
-  auto i = 1;
-  auto len = 0;
-  for (len = split.size(); i < len; i += 2) {
+  int i = 1;
+  int len;
+  std::string colorCode = "^";
```

Comment:
nit: Could be a oneliner: `std::string colorCode = "^" + static_cast<char>(color);`